aten/src/ATen/Context.h

```diff
@@ -114,12 +114,37 @@ class CAFFE2_API Context {
   void setDeterministicCuDNN(bool);
   bool deterministic() const;
   void setDeterministic(bool);
+
+  // This method should be called by operations that act nondeterministically, so that they
```

Comment:
Let's start by describing what the function actually does first, then show an example of how to use it. Maybe something like, "throws an a RuntimeError if `Context::deterministic()` is true." This should be called before performing a nondeterministic operation like this:

<example description>
<NOTE>
alertNotDeterministic(<caller>)
<nondeterministic operations here>

Reply:
Done